public/index.html

```diff
@@ -26,6 +26,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@crayne we had actually decided to remove font awesome last week to improve page load since we were only using it for a Facebook icon. Can you find minified svg or jpg images and use those instead?

Reply:
Page load is much faster now since chunked loading was implemented.  So is it still necessary to remove font awesome?

Reply:
what's the size diff of font awesome vs. small icons for each social platform? I'd vote for whatever is smallest since we aren't using font-awesome anywhere else at this point

Reply:
Individual icons are much smaller so I'll use them